src/js/base/api.js

```diff
@@ -1166,8 +1166,8 @@ ripe.Ripe.prototype._partsToPartsM = function(parts) {
     return partsM;
 };
 
-ripe.Ripe.prototype._partsMToQuery = function(partsM, sort = true) {
-    const queryL = [];
+ripe.Ripe.prototype._partsMToTriplets = function(partsM, sort = true) {

```

Comment:
Good idea (method re-usage), we need to back-port this solution to the Python version

Reply:
Same method re-usage for ripe-core :arrow_right: https://github.com/ripe-tech/ripe-core/pull/4500